res/skins/LateNight/samplers_unit_1-8_big.xml

```diff
@@ -0,0 +1,112 @@
+<Template>
+  <WidgetGroup>
+    <Layout>horizontal</Layout>
+    <SizePolicy>me,max</SizePolicy>
+    <Children>
+
+      <WidgetGroup><Size>2f,1min</Size></WidgetGroup>
+
+      <!-- Don't know why, but this button has to be inverted..-->
+      <PushButton>
+          <TooltipId>show_samplers</TooltipId>
+          <ObjectName>Samplerexpandbutton</ObjectName>
+          <Size>min,min</Size>
+          <NumberStates>2</NumberStates>
+          <State>
+            <Number>1</Number>
+            <Pressed>skin:/buttons/btn_samplerExpand.svg</Pressed>
+            <Unpressed>skin:/buttons/btn_samplerExpand.svg</Unpressed>
+          </State>
+          <State>
+            <Number>0</Number>
+            <Pressed>skin:/buttons/btn_samplerCollapse.svg</Pressed>
+            <Unpressed>skin:/buttons/btn_samplerCollapse.svg</Unpressed>
+          </State>
+          <Connection>
+            <ConfigKey>[Samplers],show_big</ConfigKey>
+            <ButtonState>LeftButton</ButtonState>
+          </Connection>
+      </PushButton>
+
+      <WidgetGroup><Size>2f,1min</Size></WidgetGroup>
+
+      <WidgetGroup>
+      <Layout>vertical</Layout>
+      <SizePolicy>min,min</SizePolicy>
+      <Children>
+
+        <!-- First row -->
+        <WidgetGroup>
+        <ObjectName>SamplerTextSmall</ObjectName>
```

Comment:
here & below: please keep an eye on the indentations

Reply:
ok, done